packages/core/extension.package.json

```diff
@@ -38,5 +39,8 @@
       "frontend": "lib/browser/clipboard/browser-clipboard-module",
       "frontendElectron": "lib/electron-browser/clipboard/electron-clipboard-module"
     }
-  ]
+  ],
+  "scripts": {
+    "docs": "typedoc --tsconfig compile.tsconfig.json --options tdoptions.json ."
```

Comment:
We can add this to the extension generator. It is entirely ours and not meant to be used by external extension developers